### PR TITLE
Checkpoint-02 (Ian Hollands)

### PR DESCRIPTION
**Pull request by ihollands**
Comfort: 5
Completeness: 4

### CODE REVIEW

**Comment by ihollands** [comment]
Javascript Part I didn't get commited properly...probably my fault. The correct steps were taken and are in the script.js file, however

File: index.html [hunk 1]
```diff
@@ -4,12 +4,14 @@
   <meta charset="UTF-8">
   <title>Checkpoint 02</title>
   <link rel="stylesheet" href="style.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
```
Comment (amaseda) [inline index.html:7]: :+1:

File: index.html [hunk 2]
```diff
@@ -4,12 +4,14 @@
...
 </head>
 <body>
   <div id="stoplight">
-    <div id="red" class="bulb active"></div>
+    <div id="red" class="bulb"></div>
     <div id="yellow" class="bulb"></div>
     <div id="green" class="bulb"></div>
   </div>
+  <script src="stoplight.js"></script>
```
Comment (amaseda) [inline index.html:15]: :+1:

File: script.js [hunk 3]
```diff
@@ -1,11 +1,18 @@
 // Create an array called `colors` that contains three colors.
-
+var colors = ["red", "green", "blue"];
```
Comment (amaseda) [inline script.js:2]: :+1:

File: script.js [hunk 4]
```diff
@@ -1,11 +1,18 @@
...
 // Log the last item in the array to the console.
-
+console.log(colors[2]);
```
Comment (amaseda) [inline script.js:4]: :+1:

File: script.js [hunk 5]
```diff
@@ -1,11 +1,18 @@
...
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
-
+for (var i = 0; i < 3; i++) {
+  console.log("My favorite color is " + colors[i]);
+}
```
Comment (amaseda) [inline script.js:8]: :+1:

File: script.js [hunk 6]
```diff
@@ -1,11 +1,18 @@
...
 // Create an `instructor` object that contains three key-value pairs.
-
+var instructor = {
+  topic: "web development",
+  years: 4,
+  computer: "Mac"
+}
```
Comment (amaseda) [inline script.js:14]: :+1:

File: script.js [hunk 7]
```diff
@@ -1,11 +1,18 @@
...
 // Add a `facial-hair` property to `instructor`.
-
+instructor["facial-hair"] = true;
```
Comment (amaseda) [inline script.js:16]: :+1:

File: script.js [hunk 8]
```diff
@@ -1,11 +1,18 @@
...
 // Log the value of the `facial-hair` property to the console.
+console.log(instructor["facial-hair"])
```
Comment (amaseda) [inline script.js:18]: :+1:

File: stoplight.js [hunk 9]
```diff
@@ -0,0 +1,29 @@
+var redBulb = $("#red");
+var yellowBulb = $("#yellow");
+var greenBulb = $("#green");
+
+activateRed = function() {
+  redBulb.toggleClass("active")
```
Comment (amaseda) [inline stoplight.js:6]: :+1:

File: stoplight.js [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+var activateYellow = function() {
+  yellowBulb.toggleClass("active");
+}
+
+var yellowBehavior = function() {
+  setTimeout(redAfterYellow, 2000);
+  setTimeout(activateYellow, 2000)
+}
```
Comment (amaseda) [inline stoplight.js:20]: :+1:

File: stoplight.js [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+
+var activateGreen = function() {
+  greenBulb.toggleClass("active")
+}
+
+redBulb.on("click", activateRed);
+yellowBulb.on("click", yellowBehavior);
+yellowBulb.on("click", activateYellow);
+greenBulb.on("click", activateGreen);
```
Comment (amaseda) [inline stoplight.js:29]: You could implement the one-light-at-a-time functionality using `.toggleClass` and/or `.siblings` to turn off other lights.

**Pull request closed without merging**